src/planscape/plan/tests.py

```diff
@@ -1311,17 +1311,20 @@ def test_get_scenario_ok(self):
             {'id': self.scenario.pk},
             content_type="application/json")
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(response.json()['owner'], self.user.pk)
-        self.assertEqual(response.json()['project'], self.project.pk)
-        self.assertEqual(response.json()['plan'], self.plan.pk)
-        self.assertEqual(response.json()['notes'], 'my note')
-        self.assertEqual(response.json()['priorities'], {
+        scenario = response.json()
+        self.assertEqual(scenario['owner'], self.user.pk)
```

Comment:
is project config info also being returned in scenario response? (max_road_distance, max budget, etc)

Reply:
@lattemilk it is, i can make that evident in the tests too